docs/sources/azurestoragequeue.md

```diff
@@ -2,6 +2,10 @@
 
 This event source is meant to be used as a Container Source with a Knative cluster to consume messages from a Azure Storage Queue and send them to a Knative service/function.
 
+## Prerequisite(s)
+
+- Azure Storage Queue
+
```

Comment:
Same as other cases, this file is not linked to the index. We should clean these files to avoid people working on them.

Reply:
@odacremolbap are you meaning to delete the whole file?

Reply:
yes delete